### DISCUSSION TITLE
Feature request: optional text mode of cards.

### DISCUSSION THREAD

**Discussion by FreeLikeGNU** [bafolts/terraforming-mars#450] [Ideas]
As much as I love the card representations as graphics, it could be nice to have the option of cards as a simple detailed list or table to make the UI more compact and less demanding on the rendering device.

If this ever leads to a Curses version of TM that runs in a terminal, I'm totally happy with that too ;-)

TAGS | TITLE | REQ | CONTENT | EFFECTS
--------|------------|-------|------------------|---------------
S | **helion** ||You start with 3 heat production and 42 MC. |**_Effect: Your may use heat as MC. You may not use MC as heat._**
E,B | **Deep Well Heating** || Increase your Energy Production 1 step.
C,B | **Capital**| _Requires 4 ocean tiles._ |Place this tile. Decrease your Energy production 2 steps and increase your MC production 5 steps. |**_1 ADDITIONAL VP FOR EACH OCEAN TILE ADJACENT TO THIS CITY TILE._**

**Comment by nwai90** [comment]
As a longtime player I also like the idea of compactness when playing on smaller devices, albeit in a different form - to represent every card with only iconography and no text. This could be a nice option for experienced players but it's also a major piece of work haha

<img width="227" alt="Screenshot 2020-06-16 at 9 27 06 PM" src="https://user-images.githubusercontent.com/2408094/84780497-3bf77c00-b018-11ea-8893-8d7838e1328e.png">
<img width="226" alt="Screenshot 2020-06-16 at 9 16 42 PM" src="https://user-images.githubusercontent.com/2408094/84780505-3d28a900-b018-11ea-9d91-e0a62324a866.png">
<img width="226" alt="Screenshot 2020-06-16 at 8 59 48 PM" src="https://user-images.githubusercontent.com/2408094/84780514-3e59d600-b018-11ea-82e6-67a7a9b940bc.png">

**Comment by funkybaboon** [comment]
Once we decide how the new card templates look like (https://github.com/bafolts/terraforming-mars/issues/412) we can probably add support for a text only / icons only version of the cards.
